assets/components/Wrapper.jsx

```diff
@@ -158,15 +158,28 @@ const styles = theme => ({
 });
 @withStyles(styles)
 export default class Bar extends Component {
-  state = {
-    open: false,
-  };
+  constructor(props) {
```

Comment:
Very bad. Everything in this file should be completely changed.
1. Think of it – you set the drawer state *open* when the localStorage variable *isDrawerClosed* is true?? Not vice-versa?
1. You should not save any information into localStorage during construction. If no information is found, jut count it as false value.
1. Make everything shorter. The construction can be this simple:
```
this.state = {
  open: !localStorage.getItem("isDrawerClosed")
}
```
And, when you open the drawer back, you can just remove this flag from the localStorage instead of setting "false" string.

Reply:
Fixed.